_posts/2021-07-15-AI-fairness/AI_fairness_metrics.Rmd

```diff
@@ -0,0 +1,415 @@
+author:
+  - name: Sigrid Keydana
+    affiliation: RStudio
+    affiliation_url: https://www.rstudio.com/
+slug: keydanafairnessmetrics
+date: 07-15-2021
+bibliography: bibliography.bib
+categories:
+  - R
+  - Concepts
+  - Meta
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+preview: images/preview.jpg
+editor_options: 
+  markdown: 
+    wrap: 72
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, eval = FALSE)
+```
+
+If you use deep learning for unsupervised part-of-speech tagging of
+Sanskrit [^1], or knowledge discovery in physics [^2], you probably
+don't need to worry about model fairness. If you're a data scientist
+working at a place where decisions are made about *people*, however, or
+an academic researching models that will be used to such ends, chances
+are that you've already been thinking about this topic. Or feeling that
+you should. And thinking about this is hard.
+
+[^1]: @Srivastava
+
+[^2]: @Cranmer
+
+It is hard for several reasons. In this text, I will go into *just one*.
+
+## The forest for the trees
+
+Nowadays, it is hard to find a modeling framework that does *not*
+include functionality to assess fairness. (Or is at least planning to.)
+And the terminology sounds so familiar, as well: "calibration",
+"predictive parity", "equal true [false] positive rate"... It almost
+seems as though we could just take the metrics we make use of anyway
+(recall or precision, say), test for equality across groups, and that's
+it. Let's assume, for a second, it really were that simple. Then the
+question still is: Which metrics, exactly, do we choose?
+
+So in reality things are *not* simple. And it gets worse. For very good
+reasons, there is a close connection, in the ML fairness literature, to
+concepts that are primarily treated in other disciplines, such as the
+legal sciences: *discrimination* and *disparate impact* (both not being
+far from yet another statistical concept, *statistical parity*).
+Statistical parity means that if we have a classifier, say to decide
+whom to hire, it should result in as many applicants from the
+disadvantaged group (e.g., Black people) being hired as from the
+advantaged one(s). But that is quite a different requirement from, say,
+equal true/false positive rates!
+
+So despite all that abundance of software, guides, and decision trees,
+even: This is not a simple, technical decision. It is, in fact, a
```

Comment:
```suggestion
even, this is not a simple, technical decision. It is, in fact, a
```

Reply:
Still too stubborn to get rid of this type of colons ;-))) ( I know you said in English that's not rather uncommon, but ...) Maybe I'll get there some time ...